scripts/update-npm-tag.mjs

```diff
@@ -0,0 +1,53 @@
+import shell from 'shelljs';
+import { getPackageInfos } from "workspace-tools";
+
+function getPublicPackagesNames() {
+  const packages = getPackageInfos(import.meta.resolve('..'));
```

Comment:
Just for me to follow: what does this return? A list of packages in the repo?

Reply:
Yes, this returns an array with the parsed `package.json` files of all this repository packages.